src/edu/ucsb/cs56/projects/games/states_quiz/HighscoreFrame.java

```diff
@@ -0,0 +1,43 @@
+package edu.ucsb.cs56.projects.games.states_quiz;
+
+import javax.swing.JFrame;
+import javax.swing.JTextArea;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.File;
+
+public class HighscoreFrame extends JFrame {
```

Comment:
So I did some research and I think the way to make the program wait for this guy to close is to make it inherit from `JDialog` instead of `JFrame` and set `modal` to `true` when you call `super`...
 - https://stackoverflow.com/questions/7855227/wait-for-jdialog-to-close
 - https://docs.oracle.com/javase/9/docs/api/javax/swing/JDialog.html#JDialog-java.awt.Frame-java.lang.String-boolean-